ouroboros-consensus/src/Ouroboros/Consensus/Node.hs

```diff
@@ -284,40 +289,93 @@ initInternalState NodeParams {..} = do
           (tracePrefix "ChainDB" Nothing)
           cfg chainDB getCandidates blockFetchSize blockMatchesHeader btime
 
-        nrChainSyncClient :: up -> Consensus ChainSyncClient hdr m
-        nrChainSyncClient up = chainSyncClient
-          (tracePrefix "CSClient" (Just up))
-          cfg
-          btime
-          maxClockSkew
-          (ChainDB.getCurrentChain chainDB)
-          (ChainDB.getCurrentLedger chainDB)
-          varCandidates
-          up
 
-        nrChainSyncServer :: ChainSyncServer hdr (Point hdr) m ()
-        nrChainSyncServer =
-          chainSyncServer (tracePrefix "CSServer" Nothing) chainDB
-
-        nrBlockFetchClient :: BlockFetchClient hdr blk m ()
-        nrBlockFetchClient = blockFetchClient
-          -- Note the tracer for the changes in the fetch client state
-          -- is passed to the blockFetchLogic.
-          -- The message level tracer is passed to runPipelinedPeer.
-
-        nrBlockFetchServer :: BlockFetchServer blk m ()
-        nrBlockFetchServer =
-          blockFetchServer (tracePrefix "BFServer" Nothing) chainDB
-
-        nrFetchClientRegistry = fetchClientRegistry
-
-        networkRequires :: NetworkRequires m up blk hdr
-        networkRequires = NetworkRequires {..}
+        responderApp :: NodeToNodeProtocols
```

Comment:
This responder/initiator stuff doesn't really belong in `initInternalState`, it should move to `initNetworkLayer`.

Reply:
If we reinstate the `NetworkRequires` abstraction (see next comment), these `nr-` definitions can be brought back.